src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -720,19 +720,9 @@ public static int getItemCountInProvider(@NotNull final ICapabilityProvider prov
     public static int hasBuildingEnoughElseCount(@NotNull final IBuilding provider, @NotNull final ItemStorage stack, final int count)
```

Comment:
There are several places that assume they will always get count from this, the MAXINT variant return being rare. 
Specifically, the AbstractBuildingWorker and the Dyer both use this function to figure out which ingredient is in largest supply.  Do we actually need the MAXINT result?

Reply:
This method is kinda greedy, meaning that the moment we found enough we return. Thus, the returned count != the actually available count but just > the input count. Thus, if return is maxInt we know we got enough, else we know we got just as much as the return value specifies.